spaces/7add/index.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "HOLDTOWIN",
+  "network": "1",
+  "symbol": "7ADD",
+  "strategies": [
+    {
+      "name": "erc20-balance-of",
+      "params": {
+        "address": "0xed36482c7f8e5850e91ac0cf6bf2130a1aa2df92",
+        "symbol": "7ADD",
+        "decimals": 18
+      }
+    }
+  ],
+  "members": [
+    "0x1a6c95c161b0f4159a65371ed1113bc1f6257add"
```

You will miss a comma after this line.
```suggestion
    "0x1a6c95c161b0f4159a65371ed1113bc1f6257add",
```